spec-0000/chart.md

```diff
@@ -4,9 +4,9 @@ axisFormat %m / %Y
 title Support Window
 
 section python
-3.11 : 2022-10-24,2025-10-23
-3.12 : 2023-10-02,2026-10-01
-3.13 : 2024-10-07,2027-10-07
+3.11 : 2022-10-24,2025-10-01
+3.12 : 2023-10-02,2026-10-07
+3.13 : 2024-10-07,2026-10-07
```

Comment:
there must be a bug here somewhere, we don't cover the case of what happens when there is no next release yet. IMO with the no logic there should not be a drop date.